homeassistant/components/overkiz/executor.py

```diff
@@ -95,6 +95,24 @@ async def async_execute_command(self, command_name: str, *args: Any) -> None:
 
         await self.coordinator.async_refresh()
 
+    async def async_execute_commands(self, commands: list[list | str]) -> None:
```

Comment:
Type is not correct? Second parameter should not be a string probably?

Reply:
Second parameter is an array of all commands we want to execute.  
Each command could be :  
- a string to execute a command without parameter
- a list to execute a command with first element of array is the command name, second parameter is a list of parameter for the command

See https://github.com/home-assistant/core/pull/78659/files#diff-91dd0349008bdb0dc5b517ae3e72ccd729ca1800ec222dbfcddef82040775942R93 for an example usage

Reply:
I could add option to pass a Command, but you asked to not do so here https://github.com/home-assistant/core/pull/78659#discussion_r973760583 in order to be consistent with  async_execute_command.
Maybe async_execute_command should allow Command as well too as only parameter?

Reply:
Hi @iMicknl, could you let me know what you me to do in order to be able to accept this PR?

Reply:
@nyroDev I did not have time to fully review this yet. And next to me, a core contributor will also have to review the changes. So be aware that this will take quite some time, most PR's are merged in a few months only.

You can help speeding this up by reviewing other PR's, to reduce the workload of core reviewers.

Reply:
@iMicknl I'm pretty new to HA (and to python too...), so not very confident to comment on other PR.
I try to check some, but cannot find places where I could help.

I noticed pyoverkiz did call `execute_commands` when calling `execute_command`
So I updated my code to do the same on our side.
I also updated the code to reflect all elements that was here for single command:
RTS parameter, saving execution id and canceling them

After this PR is merged, I could update some others elements where multiple commands are done one after another.

Reply:
Closing this thread in favor of splitting to another PR #79576